src/ring_theory/ideal/operations.lean

```diff
@@ -535,18 +535,26 @@ begin
       multiset.prod_cons, one_eq_top, exists_eq_left, multiset.prod_zero,
       multiset.map_zero, multiset.mem_singleton] },
```

Comment:
These `map` lemmas in the simp only are no longer needed, there's no `map` in the goal any more.